lib/backend/syncersv1/updateprocessors/rules_test.go

```diff
@@ -296,7 +298,38 @@ var _ = Describe("Test the Rules Conversion Functions", func() {
 		})
 
 		By("generating the correct destination selector", func() {
-			Expect(rulev1.SrcSelector).To(Equal(e))
+			Expect(rulev1.DstSelector).To(Equal(e))
+		})
+	})
+
+	It("should parse a serviceaccount selector", func() {
```

Comment:
I think we also need a few other tests here:

- should parse a rule with both a serviceaccount and a namespace selector.
- should parse a rule with both a serviceaccount and wep selector.
- should parse a rule with all three.